build.sbt

```diff
@@ -197,6 +197,8 @@ lazy val mimaSettings = Def.settings(
       .exclude[DirectMissingMethodProblem]("com.twitter.algebird.util.summer.HeavyHittersPercent.apply"),
     ProblemFilters.exclude[DirectMissingMethodProblem]("com.twitter.algebird.util.summer.BufferSize.apply"),
     ProblemFilters.exclude[DirectMissingMethodProblem]("com.twitter.algebird.util.summer.Compact.apply"),
+    ProblemFilters.exclude[DirectMissingMethodProblem]("com.twitter.algebird.MinAggregator.ord"),
```

Comment:
I guess it's backwards compat as long as people don't call `aggr.ord`?

Reply:
yeah right, with the previous syntax that was totally possible;

Reply:
I’m nervous about each of these. I think we should be very reluctant to risk binary incompatibility to clean up warts.

Reply:
@johnynek yeah those change were reverted in 2nd commit of the PR. The squashed commit doesn't have any MiMa changes.

Reply:
Ahh. Right. Good point.  Thanks.

Reply:
Yup ended up achieving the same thing; @johnynek was the access to `ord` intended by design?

Reply:
I see no reason to hide it, you can reconstruct it from max or min:

```
def lteq(a: A, b: A) = max(a, b) == b
```